portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random travel destination and image of destination to the page.
  */
 function addRandomDestination() {
   const placesToVisit = 
-      ['Tokyo', 'London', 'Rome', 'New Amsterdam', 'Taipei', 'Sichuan'];
+      ['Tokyo', 'London', 'Rome', 'Amsterdam', 'Taipei', 'Sichuan'];
 
-  // Pick a random travel destination.
+  const visitPics = {
+    'Tokyo': 'https://cdn.pixabay.com/photo/2020/01/15/19/03/shinjuku-4768674_960_720.jpg',
+    'London': 'https://c1.wallpaperflare.com/preview/271/1022/559/westminster-palace-london-city.jpg',
+    'Rome': 'https://cdn.pixabay.com/photo/2018/07/20/14/02/rome-3550739_960_720.jpg', 
+    'Amsterdam': 'https://upload.wikimedia.org/wikipedia/commons/e/e8/Amsterdam_De_Wallen_6.jpg', 
+    'Taipei': 'https://upload.wikimedia.org/wikipedia/commons/c/cc/Taipei_skyline_cityscape_at_night_with_full_moon.jpg', 
+    'Sichuan': 'https://www.goodfreephotos.com/albums/china/sichuan/other/Guangfu-pavilion-with-summit-visible-in- \
```

Comment:
Does this work? feel free to just say it does... but gross if it does. Also this is against the [style guide](https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations)